docs/documentation/features.md

```diff
@@ -198,8 +196,9 @@ in status) to handle generation filtering in memory. Thus, if an event is receiv
 resource is compared with the resource in the cache.
 
 Note that the **first approach has significant benefits** in the situation when the operator is restarted and there is
-no cached resource anymore. In case two this leads to a reconciliation of every resource, event if the resource is not
-changed while the operator was not running.
+no cached resource anymore. In case two this leads to a reconciliation of every resource in all cases, 
```

Comment:
Would have been better to do these changes in a separate PR.

Reply:
it would...will extract it.

Reply:
No need, just for future reference ;)

Reply:
hehe, too late :)